coalib/bears/Bear.py

```diff
@@ -267,6 +267,9 @@ def __json__(cls):
         Override JSON export of ``Bear`` object.
         """
         _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
+        _dict = {key: value for key, value in _dict.items()
```

Comment:
just use `get_public_members(cls).items()`, otherwise you kind of change the meaning of a variable which is worse then just making a new variable